client/src/main/java/io/pravega/client/control/impl/ModelHelper.java

```diff
@@ -324,6 +325,21 @@ public static final StreamConfig decode(String scope, String streamName, final S
         return builder.build();
     }
 
+    /**
+     * Converts StreamConfiguration into StreamConfig.
+     *
+     * @param scope the stream's scope
+     * @param streamName The Stream Name
+     * @param subscriberId Id of the subscriber for this stream.
+     * @return StreamSubscriberInfo instance.
+     */
+    public static final StreamSubscriberInfo decode(String scope, String streamName, final String subscriberId) {
+        Preconditions.checkNotNull(subscriberId, "subscriber");
```

Comment:
nit: preconditions for scope and stream as well.

Reply:
sure, will add these.

Reply:
done